include/cpp2util.h

```diff
@@ -1114,7 +1114,9 @@ inline auto to_string(...) -> std::string {
     return "(customize me - no cpp2::to_string overload exists for this type)";
 }
 
-inline auto to_string(std::any const&) -> std::string {
+template<typename T>
+    requires std::is_same_v<T, std::any>
+inline auto to_string(T const&) -> std::string {
```

Comment:
```suggestion
inline auto to_string(std::same_as<std::any> auto const&) -> std::string {
```

Reply:
Oh, I forgot that we could use concepts in this context. I need to use to it. I am correcting it.

Reply:
I pushed the corrected version. Thank you, @JohelEGP, for pointing that out!